test/trento/domain/sap_system/sap_system_test.exs

```diff
@@ -26,7 +26,138 @@ defmodule Trento.SapSystemTest do
   alias Trento.Domain.SapSystem
 
   describe "SAP System registration" do
-    test "should create an incomplete SAP system aggregate and register a database instance" do
+    test "should fail when a sap system not exists and the database instance has Secondary role with system replication ACTIVE" do
```

Comment:
`should discard sap system discovery messages from a Secondary role host if the system is not registered`

or at least change `system not exists` by `system which does not exist`

Reply:
fixed